templates/kv.tmpl

```diff
@@ -19,6 +19,14 @@
     sticky = {{$sticky}}
 {{end}}
 
+{{$healthCheck := Get "" . "/healthcheck/" "path"}}
+{{$interval := Get "30s" . "/healthcheck/" "interval"}}
+{{with $healthCheck}}
```

Comment:
I followed the same format as the load balancer line above (where `{{$sticky := Get "false" . "/loadbalancer/" "sticky"}}` is outside the `with`). I can move it inside the `{{with $healthCheck}}` if you want.